packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -543,7 +543,15 @@ class FlutterCommandRunner extends CommandRunner<void> {
     // Check that the flutter running is that same as the one referenced in the pubspec.
     if (fs.isFileSync(kPackagesFileName)) {
       final PackageMap packageMap = PackageMap(kPackagesFileName);
-      final Uri flutterUri = packageMap.map['flutter'];
+      Uri flutterUri;
+      try {
+        flutterUri = packageMap.map['flutter'];
+      } on FormatException {
+        // We're not quie sure why this can happen, perhaps the user
```

Comment:
nit: would it make sense to add the try/catch to https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/dart/package_map.dart#L37?

Reply:
Probably not, since there are places where we should crash if we can't read it. But this check runs before pub get, which should correct the .packages file

Reply:
nit: quie -> quite

Is there any user action to take here?  Should we print something, even just a `printTrace` saying that the `.packages` file was malformed and we couldn't use it where we expected to be able to?

Reply:
done